experimental/oak_functions_loader_linux_vsock/src/main.rs

```diff
@@ -53,14 +53,17 @@ pub struct Opt {
 // Connect to the file descriptor created by Bedebox using vsock.
 pub fn create_vsock_stream(file_descriptor: RawFd) -> anyhow::Result<VsockStream> {
     let stream = unsafe { VsockStream::from_raw_fd(file_descriptor) };
+    // Blocking is set in order to not return an error when the host hasn't written anything yet.
+    stream.set_nonblocking(false)
+        .map_err(|error| anyhow!("Couldn't set socket into blocking mode: {}", error))?;
     Ok(stream)
 }
 
 fn main() {
     let opt = Opt::parse();
 
     let stream = create_vsock_stream(opt.file_descriptor).expect("Couldn't create channel");
-    info!(
+    println!(
```

Comment:
It seems fine to stick to the logger though?

Reply:
The internal sandbox currently couldn't set up the `RUST_LOG` environment variable